front-end/src/components/analysis-page/AnalysisPage.jsx

```diff
@@ -50,6 +50,15 @@ export class AnalysisPage extends Component {
           >
             <div className="flex gap-y-10 w-full flex-col h-full">
               <div className= "w-full bg-gray-100 pb-2">
+                <div
+                  className="absolute m-2 mx-auto px-3"
+                  data-testid="intent-menu-button"
+                  role="button"
+                  onClick={this.openIntentMenu}
+                  aria-label="Intent Menu Button"
+                >
+                    <Hamburger />
+                </div>
```

Comment:
Maybe use an `<aside>` instead of a div

Reply:
@emilylwan Sorry wrong semantic. Try using a <nav> instead of a <div> to wrap the hamburger (or wrap the button that contains the hamburger). Either way it adds a11y attributes for screen readers.

Reply:
The text for the code isn't showing up. Which one do you mean?

Reply:
Sorry should be fixed.